### PR TITLE
First test cases

### PR DESCRIPTION
**Pull request by chuwy**
Single header doesn't get parsed though - seems like a bug in Soda - haven't looked at it yet.

### CODE REVIEW

**Review by algebraic-dev — APPROVED** [review]

**Comment by algebraic-sofia** [comment]
> Single header doesn't get parsed though - seems like a bug in Soda - haven't looked at it yet.

I'm going to rewrite the entire Soda library so I think that it's better just to not parse anything right now.

File: Tests/Framework.lean
Comment (samueldurantes) [inline Tests/Framework.lean:1]: I think that we could move this code to [Specs.lean](https://github.com/axiomed/Specs.lean) temporarily, so we don't need to declare this in all the other projects.

**Review by ssdurante — APPROVED** [review]

File: Tests/Framework.lean
Comment (samueldurantes) [reply Tests/Framework.lean:1]: What do you think @algebraic-sofia?

Comment (algebraic-sofia) [reply Tests/Framework.lean:1]: I'm just adjusting some things, I think that Specs.Lean will be ready to use in a few hours :)

Comment (chuwy) [reply Tests/Framework.lean:1]: I'll rewrite the tests to use `Specs.lean`

**Comment by samueldurantes** [comment]
Does this pull request still make sense?

Should we close? cc @algebraic-sofia

**Pull request closed without merging**